google/cloud/vision/entity.py

```diff
@@ -55,7 +55,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = []
+        if 'boundingPoly' in response:
+            bounds = Bounds.from_api_repr(response['boundingPoly'])
```

Comment:
Is this part stray, or a rebase artifact?  It doesn't seem related to the rest of the PR.  Also, I don't see any new tests which exercise the missing `bounds` key case.